src/index.ts

```diff
@@ -1,36 +1,40 @@
 // tslint:disable-next-line: no-reference
 /// <reference path="../types/dom-monkeypatch.d.ts" />
 
-/* tslint:disable:no-var-requires */
-const version = require("../package.json").version;
-
 // Expose all types
-export * from "./horizon_api";
-export * from "./server_api";
+export * from './horizon';
+export * from './soroban/soroban_rpc';
 
 // stellar-sdk classes to expose
-export * from "./account_response";
-export * from "./errors";
-export { Config } from "./config";
-export { Server } from "./server";
+export * from './errors';
+export { Config } from './config';
 export {
   FederationServer,
   FEDERATION_RESPONSE_MAX_SIZE
-} from "./federation_server";
+} from './federation/federation_server';
 export {
   StellarTomlResolver,
   STELLAR_TOML_MAX_SIZE
-} from "./stellar_toml_resolver";
+} from './federation/stellar_toml_resolver';
+
 export {
```

Comment:
since a `Horizon` namespace is exported earlier by `./horizon` module , would it benefit organizationally, to export these other couple horizon types under that namespace also rather than having them at top level of lib? similar thought for soroban rpc types exported here could be pushed under`SorobanRpc` namespace, and exported from './soroban/soroban_rpc' module instead?

Reply:
this makes a lot of sense - I'll have a think about it and try to sketch it out